util/networkutil.py

```diff
@@ -28,6 +28,33 @@
 
 DEV_CHAINID = _NETWORK_TO_CHAINID["development"]
 
+CHAIN_ADDRS = {
+    8996: "0xdev",
+    1: "0xeth",
+    3: "0xropsten",
+    4: "0xrinkeby",
+    56: "0xbnb",
+    137: "0xpolygon",
+    246: "0xenergyweb",
+    1285: "0xmoonriver",
+    1287: "0xmoonbase",
+    80001: "0xmumbai",
+}
```

Comment:
We can't have this, big DRY violation due to overlap with `_CHAINID_TO_NETWORK`. If you want to have this dict, auto-compute it as a function of `_CHAINID_TO_NETWORK`.

Reply:
What if we create a list in the code, something like:

```python
_CHAIN_INFO = [
#chainid, name, native_token
8996, "development", "ETH",
1, "mainnet", "ETH",
3, "ropsten", "ETH",
...
137, "polygon", "MATIC",
...
]

Then, other caches or dicts can be computed from this.

Less error prone, because all data is in one place.